src/extlibs/enlsvg/Pathfinding/LineOfSightScanner.h

```diff
@@ -76,37 +76,37 @@ namespace Pathfinding
                                 : left_down_extents[(y + 1) * extents_size_x + x_L];
         }
 
-        inline int leftDownExtent(int xL, int y) const
+        inline int leftDownExtent(int x_L, int y) const
```

Comment:
I think the `L` in these variables should be made small. But to also make it a bit more readable, could you replace the instances of `x_L` to be `x_left` and similarly for `x_R`?